.github/workflows/ci-build.yml

```diff
@@ -2,71 +2,68 @@ name: CI Build
 
 on:
   - push
+  - pull_request
```

Comment:
```suggestion
  push:
    branches:
      - main
  pull_request:
```

Reply:
Nice catch! I think this also missed in project-keeper.

Reply:
@kaklakariada fixed it in 1.3.4